src/components/dialogs/create-case-dialog.js

```diff
@@ -85,7 +85,15 @@ export function CreateCaseDialog({ onClose, open }) {
         })
             .then()
             .catch((message) => {
-                snackbarMessage(message, 'caseCreationError', { name });
+                message.includes(UNPROCESSABLE_ENTITY_STATUS)
```

Comment:
We are still not relying on the status code of the response, which makes the backend modification useless here
Is it possible to fix it ?

Reply:
yes